stream_test.go

```diff
@@ -0,0 +1,60 @@
+package openssl
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+)
+
+func NoError(t *testing.T, err error) {
+	if err != nil {
+		t.Error(err)
+		t.FailNow()
+	}
+}
+
+func TestReader(t *testing.T) {
+	o := New()
+
+	pass := "abcd"
+	plaintext := []byte("123abc,./vvvczcekdewfeojdosndsdlsndlncnepcnodcnviorf409eofnvkdfvjfvdsoijvo")
+
+	data, err := o.EncryptBinaryBytes(pass, plaintext, BytesToKeyMD5)
+	NoError(t, err)
+
+	buf := bytes.NewBuffer(nil)
+	es := NewWriter(pass, buf, BytesToKeyMD5)
+	_, err = es.Write(plaintext)
+	NoError(t, err)
+	NoError(t, es.Flush())
```

Comment:
Why does the `TestReader` test function utilize the `NewWriter`? It should take encrypted data (`data`) and read from it comparing the read data to the `plaintext`.